cnd/src/db/mod.rs

```diff
@@ -0,0 +1,162 @@
+mod models;
+mod schema;
+
+pub use crate::db::models::Metadata;
+
+use crate::{
+    db::models::NewMetadata,
```

Comment:
Does it though? For me, aliases cause major headache usually because you cannot find the type when you are looking for it ..

Reply:
Arg, wrong line, there are 2 types of `Metadata`.
I don't have your headache but I'd be happy to just keep the path so that they can be differentiate in file.